stubs/pywin32/isapi/install.pyi

```diff
@@ -56,43 +59,43 @@ class ISAPIParameters:
 
 verbose: int
 
-def log(level, what) -> None: ...
+def log(level: int, what: str) -> None: ...
 
 class InstallationError(Exception): ...
 class ItemNotFound(InstallationError): ...
 class ConfigurationError(InstallationError): ...
 
-def FindPath(options, server, name): ...
-def LocateWebServerPath(description): ...
-def GetWebServer(description: Incomplete | None = None): ...
+def FindPath(options, server: str | bytes, name: str) -> str: ...
```

Comment:
`bytes` and `bytearray` both support `.decode()`
Relates to https://github.com/python/typeshed/issues/9006

````suggestion
def FindPath(options, server: str | bytes | bytearray, name: str) -> str: ...
```